src/App/Service/ExcelExport.php

```diff
@@ -130,6 +130,7 @@ public function generateExcelFile(LsDoc $cfDoc, array $items, array $association
             ->setCellValue('K2', $cfDoc->getAdoptionStatus())
             ->setCellValue('L2', $cfDoc->getStatusStart())
             ->setCellValue('M2', $cfDoc->getStatusEnd())
+            ->setCellValue('N2', $cfDoc->getLicence()->getLicenceText())
```

Comment:
We might need to export more than just the text (which could be very long) if we want to be able to re-import the file.

Reply:
If there is no licence set then `$cfDoc->getLicence()` will be `null`, so need to account for that here.